src/SFA.DAS.EmployerApprenticeshipsService.Application.UnitTests/Queries/SendTransferConnectionInvitationTests/WhenISendTransferConnectionInvitation.cs

```diff
@@ -77,6 +77,15 @@ public void Arrange()
             Assert.That(_response.ReceiverAccount.Id, Is.EqualTo(_receiverAccount.Id));
         }
 
+        [Test]
+        public async Task ThenSenderAccountShouldBePopulatedOnSendTransferConnectionInvitationResponse()
```

Comment:
The assertions in this test could be moved to `ThenShouldReturnSendTransferConnectionInvitationResponse`.